sdk/Lusid.Sdk.Tests/tutorials/Properties/TransactionProperties.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using Lusid.Sdk.Api;
+using Lusid.Sdk.Client;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Properties
+{
+    [TestFixture]
+    public class TransactionProperties : TutorialBase
+    {
+        private IList<string> _instrumentIds;
+        
+        [OneTimeSetUp]
+        public void OnetimeSetUp()
+        {
+            var instrumentLoader = new InstrumentLoader(_apiFactory);
+            _instrumentIds = instrumentLoader.LoadInstruments();
+        }
+
+        internal string _portfolioCode;
+        [SetUp]
+        public void SetUp()
+        {
+            var portfolioRequest = TestDataUtilities.BuildTransactionPortfolioRequest();
+            var portfolio = _transactionPortfoliosApi.CreatePortfolio(TestDataUtilities.TutorialScope, portfolioRequest);
+            _portfolioCode = portfolioRequest.Code;
+            Assert.That(portfolio?.Id.Code, Is.EqualTo(_portfolioCode));
+        }
+
+        [LusidFeature("F13-4")]
```

Comment:
we should move this to https://github.com/finbourne/lusid-sdk-csharp-preview/blob/master/sdk/Lusid.Sdk.Tests/tutorials/Ibor/Transactions.cs alongside the other transaction examples

Reply:
OK, this was originally placed in its own class and namespace to mirror Python. Now back with Transactions.cs

Reply:
In this case i think its fine. The purpose of the examples (aka tests) is to show a user how to craft a call to LUSID so the focus should be on clarify of code rather than the most efficient or being overly concise